### ISSUE TITLE
Cannot integrate RTK

### ISSUE THREAD

**Issue by Youssefeh** [xsenssupport/Xsens_MTi_ROS_Driver_and_Ntrip_Client#12]
I am failing to see any RTCM messages after running ntrip process on ros2 humble. Logs on SmartNet indicate that no connection was made when using the driver. Not sure what the issue might be. Any help would be appreciated. T
Thank you!

**Comment by xsenssupport** [comment]
Please make sure you had filled in your own credentials for the Ntrip client and after that, make sure you do 'colcon build' again.

**Comment by Youssefeh** [comment]
Yes I have done that and on the SmartNet site it indicates that a connection attempt was made, however access was denied as per the image attached here. When entering the same credentials on the windows application it connects just fine and I am able to receive RTCM messages. 
![Screenshot from 2024-11-17 15-31-06](https://github.com/user-attachments/assets/fef694f7-dca6-4b3f-8188-c203f8de20bc)

**Comment by xsenssupport** [comment]
I don't know about SmartNet, from your screenshot, it looks like your credential isn't correct and got rejected.
Do you have the ros2 topic /nmea published?

**Comment by xsenssupport** [comment]
> Yes the /nmea topic is being published

can you write an email to support@movella.com?

**Comment by xsenssupport** [comment]
> Yes the /nmea topic is being published

Hi,  
We just updated to a newer version of the NTRIP client. If you have time to test it and let us know your results, that would be great. Thanks!
